95_resources.Rmd

```diff
@@ -28,7 +28,7 @@
 [NGS Analysis Basics](http://girke.bioinformatics.ucr.edu/GEN242/tutorials/rsequences/rsequences/) provides a walk-through of the above-mentioned features with detailed examples.
 
 
-### Alternative containers for microbiome data
+### Phyloseq: an alternative container for microbiome data
```

Comment:
It is "phyloseq", not "Phyloseq" (same holds when sentence starts if it is a name that should be written with lowercase)